app/src/main/java/hm/orz/chaos114/android/androidsaddlersample/MainActivity.java

```diff
@@ -7,7 +7,8 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private boolean HOGE;
+    private boolean FOO;
```

Comment:
: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not\nhttp://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName